content/shared/tutorials/_android_push_enable_permissions.textile

```diff
@@ -1,15 +1,22 @@
-Create a sandbox application as shown below and make sure that both the @Push Admin@ and @Push Subscribe@ permissions are checked, for your API key. You can do this under the settings tab of your app in the developer dashboard.
+Next, you'll need to ensure your Ably app is correctly set up to use Push Notifications. Firstly, make sure the API key you're using has both the @Push Admin@ and @Push Subscribe@ permissions, as seen below. If not, select the *Settings* option on the API key interface and tick the permissions so they're available.
 
-<a href="/images/tutorials/android-push/sandbox-app.png" target="_blank">
-    <img src="/images/tutorials/android-push/sandbox-app.png" style="width: 100%" alt="Setting up a Sandbox app">
+<a href="/images/tutorials/push-android/adding-capabilities.png" target="_blank">
+  <img src="/images/tutorials/push-android/adding-capabilities.png" style="width: 100%" alt="Adding Push capabilities">
 </a>
 
-<a href="/images/tutorials/android-push/adding-capabilities.png" target="_blank">
-    <img src="/images/tutorials/android-push/adding-capabilities.png" style="width: 100%" alt="Adding Push capabilities">
-</a>
+These permissions correspond to the following:
+
+* "@push-subscribe@":/general/push/activate-subscribe: A client with this permission is a push target device, and it can manage registration and subscription to Push Notifications for itself. However, it won't be able to manage push registrations or channel subscriptions for any other device.
+* "@push-admin@":/general/push/admin: A client with this permission has full access to the "Push Admin API":https://www.ably.io/documentation/general/push/admin and can manage registrations and subscriptions for all devices.
+
+The @push-subscribe@ permission will be important for the Android app we'll be making, such that it can subscribe to Push Notifications. @push-admin@ is what will allow us to publish Push Notifications, and would be useful if we wanted to create a server which controlled our push devices.
```

Comment:
This first sentence is confusing is the 'it' being referred to the app or the permission? 
I'd suggest changing 'such' to 'so' or rewriting this sentence.